### ISSUE TITLE
Lnbits is not handle anymore 

### ISSUE THREAD

**Issue by sepiropht** [Dolu89/ligess#12]
Th readme is outdated, no ? In the code i don't see any connection with lnbits

**Comment by Dolu89** [comment]
True... LNBits has been removed on this refacto: https://github.com/Dolu89/ligess/commit/c8ab00b9d89b2ef0b06be625df72a599f95ee470
I don't remember this part, but I think it's an oversight. I'll add it again

**Comment by sepiropht** [comment]
OK, i will try to make a pr if i understand something
